app/src/main/java/mozilla/lockbox/adapter/AccountConfiguration.kt

```diff
@@ -0,0 +1,13 @@
+/*
```

Comment:
Prefer calling this `AccountViewModel` rather than a configuration. Configurations often have different, specific meanings, often including ids, URLs or keys.